ix-dev/community/lyrion-music-server/app.yaml

```diff
@@ -0,0 +1,39 @@
+app_version: 9.0.0
+capabilities:
+- description: LMS is able to chown files.
+  name: CHOWN
+- description: LMS is able to set group ID for it's sub-processes.
+  name: SETGID
+- description: LMS is able to set user ID for it's sub-processes.
+  name: SETUID
+categories:
+- media
+description: Lyrion Music Server - controls a wide range of Squeezebox audio players
+home: https://lyrion.org/
+host_mounts: []
+icon: https://media.sys.truenas.net/apps/lyrion-music-server/icons/icon.png
+keywords:
+- media
+- entertainment
+- music
+- streaming
+lib_version: 2.1.8
```

Comment:
can you either update the library files or changed it back ot 2.1.7 and I'll update to another PR

Reply:
Rebased to the tip of master, reran the `.github/... command`, git rm'd the 2.1.7 lib and added the 2.1.8 lib.

Hopefully that's what you need.